docs_app/content/tutorial/6-chaining-operators.md

```diff
@@ -2,13 +2,144 @@
+}
+
+const applyAllPiped = pipe(double, makeExciting, allCaps);
+
+const applyAllManually = (input: string) => allCaps(makeExciting(double(input)));
+
+console.log(applyAllPiped('more')); // "MORE AND MORE!!!"
+console.log(applyAllManually('more')); // "MORE AND MORE!!!"
+```
+
+## So What Is Pipe, Really?
+
+Pipe is a simple function that looks (basically) like this in JavaScript (The TypeScript definition gets a little hairy, so I'll just give this example in plain JS):
+
+```ts
+function pipe(...fns) {
+    return (input) => fns.reduce((prev, fn) => fn(prev), input);
+}
+
+// or a more readable version (this is identical in behavior):
+
+function pipe(...fns) {
+    return (input) => {
+        let result = input;
+        for (const fn of fns) {
+            result = fn(input);
+        }
+        return result;
+    }
+}
+```
+
+There are edge cases to cover, of course, like zero arguments, but overall this is the concept.
+
+## Piping Operators
+
+Since operators are all higher order functions that return unary functions of the shape `(source: Observable<A>) => Observable<B>`, that means they can be combined with [`pipe`](API).
+
+```ts
+import { pipe } from 'rxjs';
+import { map, filter } from 'rxjs/operators';
+
+// Here we're combining 3 operators into a single operator function
+// that is `(source: Observable<Apple>) => Observable<PeeledAppleSlice[]>`
+const processApples = pipe(
+    // `(source: Observable<Apple>) => Observable<Apple>`
+    filter(apple => apple.quality === 'GOOD'),
+    // `(source: Observable<Apple>) => Observable<PeeledApple>`
+    map(apple => createPeeledApple(apple)),
+    // `(source: Observable<PeeledApple>) => Observable<PeeledAppleSlice[]>`
+    map(peeledApple => slice(peeledApple))
+);
+```
+
+Note that `processApples` above is **NOT** an observable. It's just a function. It's a function built from three other functions that were returned by calls to [`filter`](API), [`map`](API), and another [`map`](API), in that order. You can now use it like so:
+
+```ts
+import { pipe } from 'rxjs';
+import { map, filter } from 'rxjs/operators';
+
+const apples$: Observable<Apple> = getApples();
+
+const processApples = pipe(
+    filter(apple => apple.quality === 'GOOD'),
+    map(apple => createPeeledApple(apple)),
+    map(peeledApple => slice(peeledApple))
+);
+
+processApples(apples$).subscribe(console.log);
+```
+
+This is nice, because you can reuse the operator function `processApples`, however, it's somewhat cumbersome to read, and unergonomic, so for convenience, RxJS has added a [`pipe` method](API) directly to [`Observable`](API). With the observable pipe method, the above is equivalent to:
+
+```ts
+import { map, filter } from 'rxjs/operators';
+
+const apples$: Observable<Apple> = getApples();
+
+apples$.pipe(
+    filter(apple => apple.quality === 'GOOD'),
+    map(apple => createPeeledApple(apple)),
+    map(peeledApple => slice(peeledApple))
+)
+.subscribe(console.log);
```

Comment:
I think a useful and often misunderstood part is missing here: `processApples` is a reusable operator function, made of other operators, which means that it can be used to perform the three operations (filtering mapping to a peeled apple, and mapping to slices) at once:

    apples$.pipe(processApple).subscribe(console.log);